.ci_support/linux_ppc64le_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11mpimpich.yaml

```diff
@@ -0,0 +1,49 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '11'
+c_stdlib:
+- sysroot
+c_stdlib_version:
+- '2.28'
+cdt_name:
+- conda
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge main
+cuda_compiler:
+- nvcc
+cuda_compiler_version:
+- '11.8'
+cutensor:
+- '2'
+cxx_compiler:
+- gxx
+cxx_compiler_version:
+- '11'
+docker_image:
+- quay.io/condaforge/linux-anvil-ppc64le-cuda11.8:ubi8
+mpi:
+- mpich
+mpich:
+- '4'
+openmpi:
+- '5'
+pin_run_as_build:
+  python:
+    min_pin: x.x
+    max_pin: x.x
+python:
+- 3.10.* *_cpython
+- 3.11.* *_cpython
+- 3.12.* *_cpython
+- 3.9.* *_cpython
```

Comment:
Actually starting 24.08, we require python3.10+ : https://docs.nvidia.com/cuda/cuquantum/latest/python/release-notes.html#cuquantum-python-v24-08-0

Reply:
I think this is conda forge's support matrix. There's a selector in the recipe which skips the cuquantum-python builds for python<3.10 (see output from CI and 24.11 package which is using the almost the same recipe).